packages/cashier/src/components/recent-transaction/recent-transaction.tsx

```diff
@@ -18,95 +18,18 @@ const RecentTransaction = observer(() => {
-                                        currency,
-                                    }}
-                                />
-                            </Text>
-                            {status && (
-                                <div className='cashier-recent-transaction__status'>
-                                    <span
-                                        className={classNames(
-                                            'cashier-recent-transaction__status-indicator',
-                                            `cashier-recent-transaction__status-indicator-${status.renderer}`
-                                        )}
-                                    />
-                                    <Text as='p' size='xxxs'>
-                                        {status.name}
-                                    </Text>
-                                </div>
-                            )}
-                        </div>
-                        <Text as='p' size='xxxs' color='less-prominent' line_height='s'>
-                            <Localize
-                                i18n_default_text='{{amount}} {{currency}} on {{submit_date_moment}}'
-                                values={{
-                                    amount,
-                                    currency,
-                                    submit_date_moment,
-                                }}
-                            />
-                        </Text>
-                        <div className='cashier-recent-transaction__hash-wrapper'>
-                            <div className='cashier-recent-transaction__hash'>
-                                <Text as='p' size='xxxs' line_height='s'>
-                                    <Localize i18n_default_text='Address:' />
-                                    &nbsp;
-                                </Text>
-                                <Text as='p' size='xxxs' color='red' line_height='s'>
-                                    {address_hash_display_value}
-                                </Text>
-                            </div>
-
-                            <div className='cashier-recent-transaction__hash'>
-                                <Text as='p' size='xxxs' line_height='xs'>
-                                    <Localize i18n_default_text='Transaction hash:' />
-                                    &nbsp;
-                                </Text>
-                                {status && (
-                                    <Text as='p' size='xxxs' color='red' line_height='s'>
-                                        {status.transaction_hash}
-                                    </Text>
-                                )}
-                            </div>
-                        </div>
-                    </div>
+                    <div>{children}</div>
                 </div>
                 <ButtonLink
                     to='#'
```

Comment:
Can't we use `Button` componentt? so we don't need to define `to` prop for it.

Reply:
And why we need `dc-btn--secondary` here? I think it's component-level class, right?

Reply:
@hamid-deriv This is the legacy component, We don't need to maintain it as it will be replaced by `CryptoTransactionsSideNoteRecentTransaction`.